BSB_LAN/BSB_LAN.ino

```diff
@@ -8983,6 +8906,105 @@ uint8_t pps_offset = 0;
   update_server.handleClient();
 #endif
 } // --- loop () ---
+
```

Comment:
It's a good idea to move MQTT in a separate function!
Main loop will looks more comfort for reading